content/en/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

```diff
@@ -211,3 +211,4 @@ The DEB and RPM packages shipped with the Kubernetes releases are:
 | `kubelet`    | Installs the kubelet binary in `/usr/bin` and CNI binaries in `/opt/cni/bin`. |
```

Comment:
This row also need to be modified, accordingly

Reply:
done